README.md

```diff
@@ -1,16 +1,17 @@
 # RIF Relaying Services SDK sample dApp
 
-This a basic sample dApp to showcase how users can submit transactions to the RSK blockchain paying a RelayServer for the transaction fees with tokens. You will need to connect to the dApp with Metamask but only for signing transactions with the account that controls your Smart Wallets.
+This a basic sample dApp to showcase how users can submit transactions to the RSK blockchain paying a Relay Server for the transaction fees with tokens using RIF Relay SDK. You will need to connect to the dApp with Metamask but only for signing transactions with the account that controls your Smart Wallets.
 
-See https://github.com/rsksmart/enveloping for the RIF Enveloping project.
+See [RIF Relay](https://github.com/rsksmart/rif-relay) project.
 
-## Using the dApp
 
 ## Pre-Requisites
 
-* RSKj Node Running
-* [Enveloping Contracts](https://github.com/anarancio/rif-relay-contracts) deployed.
-* [Enveloping Server](https://github.com/infuy/rif-relay-server) running.
+* [NodeJS Version v12.18.3 or higher](https://yarnpkg.com/getting-started/install).
```

Comment:
this doens't look right. @antomor mentioned that he was using node v16, did you check in the terminal which version you were running?

also, this seems to be a link to yarn. i don't think a link here was necessary, although i would have added the recommendation to us nvm instead

Reply:
still pending